### ISSUE TITLE
manifold-ctl error on Centos7

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/omnibus-manifold#6]
See end of output:

```
[root@centos7-install ~]# manifold-ctl
I don't know that command.
/opt/manifold/embedded/bin/omnibus-ctl: command (subcommand)
cleanse
  Delete *all* manifold data, and start from scratch.
uninstall
  Kill all processes and uninstall the process supervisor (data will be preserved).
upgrade
  Run migrations after a package upgrade
General Commands:
  cleanse
    Delete *all* manifold data, and start from scratch.
  help
    Print this help message.
  reconfigure
    Reconfigure the application.
  show-config
    Show the configuration that would be generated by reconfigure.
  uninstall
    Kill all processes and uninstall the process supervisor (data will be preserved).
Service Management Commands:
  graceful-kill
    Attempt a graceful stop, then SIGKILL the entire process group.
  hup
    Send the services a HUP.
  int
    Send the services an INT.
  kill
    Send the services a KILL.
  once
    Start the services if they are down. Do not restart them if they stop.
  restart
    Stop the services if they are running, then start them again.
  service-list
    List all the services (enabled services appear with a *.)
  start
    Start services if they are down, and restart them if they stop.
  status
    Show the status of all the services.
  stop
    Stop the services, and do not restart them.
  tail
    Watch the service logs of all enabled services.
  term
    Send the services a TERM.
/opt/manifold/embedded/lib/ruby/gems/2.3.0/gems/omnibus-ctl-0.3.6/lib/omnibus-ctl.rb:570:in `run': undefined method `gsub' for nil:NilClass (NoMethodError)
	from /opt/manifold/embedded/lib/ruby/gems/2.3.0/gems/omnibus-ctl-0.3.6/bin/omnibus-ctl:31:in `<top (required)>'
	from /opt/manifold/embedded/bin/omnibus-ctl:23:in `load'
	from /opt/manifold/embedded/bin/omnibus-ctl:23:in `<main>'
```

**Comment by zdavis** [comment]
Looks like it might be fixed in omnibus-ctl v0.5.0

**Comment by zdavis** [comment]
Fixed.
